tools/rna_tools/music/music.xml

```diff
@@ -0,0 +1,180 @@
+<tool id="music" name="MUSIC" version="0.1.0">
+	<description>An algorithm for identification of enriched regions at multiple scales in the read depth signals from ChIP-Seq experiments</description>
+	<requirements>
+		<requirement type="package" version="1.0.0">music</requirement>
+	</requirements>
+	<stdio>
+		<exit_code range="1:" />
+	</stdio>
+	<command><![CDATA[
+		#if str($options) == "-preprocess"
+			#if str($input_dir) == "stdin"
+				cat '$input1' |
+			#end if
+			
+			MUSIC $options $format $input_file_path $output_dir
+
+		#elif str($options) == "-sort_reads"
+			MUSIC $options $reads_dir $output_dir
+		#elif str($options) == "-remove_duplicates"
+			MUSIC $options $sorted_reads_dir $no_max_duplicates $output_dir
+		#else
+			-chip $chip_dir
+			-control $control_reads_dir
+			-mapp $mapp_dir
+			-begin_l $begin_l
+			-end_l $end_l
+			-step $step
+			-l_bin $l_bin
+
+			#if str($l_mapp)
+				-l_mapp $l_mapp
+			#end if
+
+			-mapp_thr $mapp_thr
+			-l_frag $l_frag
+			-l_c $l_c
+
+			#if str($l_p)
+				-l_p $l_p
+			#end if
+
+			-sigma $sigma
+			-gamma $gamma
+
+			#if str($q_val)
+				-q_val $q_val
+			#end if
+
+			#if str($options) == "-get_per_win_p_vals_vs_FC":
+				-l_win_min $l_win_min
+				-l_win_max $l_win_max
+				-l_win_step $l_win_step
+			#end if
+
+		#end if
+
+		mkdir SSERs
+		mkdir ERs
+		mv SSERs* SSERs/
+		mv ERs* ERs/
+
+	]]></command>
+    <inputs>
+    	<param name="options" type="select" label="Options">
+			<option value="-preprocess">Preprocess</option>
+			<option value="-sort_reads">Sort Reads</option>
```

Comment:
Oh, it should be 4 spaces and not a tab character? Ok, I got it the other way around. I'll fix it.